pkg/agent/openflow/multicluster.go

```diff
@@ -82,6 +88,7 @@ func (f *featureMulticluster) l3FwdFlowToRemoteViaTun(
 			Cookie(cookieID).
 			MatchProtocol(ipProtocol).
 			MatchDstIPNet(peerServiceCIDR).
+			Action().LoadPktMarkRange(MulticlusterPktMark, multiclusterPktMarkRange).
```

Comment:
Will geneve tunnel be replaced by wireguard totally since there is no if condition here.

Reply:
No. If there's no ip rule on the host, the packet mark will be ignored and the traffic will be forwarded by GENEVE tunnel.